Data_Engineer_KoNoHa.ipynb

```diff
@@ -0,0 +1,2775 @@
+      "source": [
+        "## Re-Columns"
+      ],
+      "metadata": {
+        "id": "Bfs7_Gn23Xoj"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Rename columns\n",
+        "df = df.rename(columns={'Price': 'Price (RM)', 'Bedrooms': 'Rooms', 'Size': 'Size (sqft)'})\n",
+        "\n",
+        "# Rearrange columns\n",
+        "df = df[['Location', 'Price (RM)', 'Rooms', 'Bathrooms', 'Car Parks', 'Property Type', 'Size (sqft)', 'Furnishing']]\n",
+        "\n",
+        "df"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 424
+        },
+        "id": "g2fu6LCy2mni",
+        "outputId": "4ca74a78-e53b-44f9-e1ef-63c7f3c50709"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "                  Location  Price (RM) Rooms  Bathrooms  Car Parks  \\\n",
+              "0                     KLCC     1250000     2        3.0        2.0   \n",
+              "2                  Dutamas     1030000     3        4.0        2.0   \n",
+              "4              Bukit Jalil      900000     4        3.0        2.0   \n",
+              "5      Taman Tun Dr Ismail     5350000     4        5.0        4.0   \n",
+              "7      Taman Tun Dr Ismail     2600000     5        4.0        4.0   \n",
+              "...                    ...         ...   ...        ...        ...   \n",
+              "53862              Seputeh      750000     3        2.0        1.0   \n",
+              "53868           KL Sentral     1400000     3        3.0        2.0   \n",
+              "53869          KL Eco City      880000     1        1.0        1.0   \n",
+              "53876         Sri Hartamas     2700000     4        6.0        3.0   \n",
+              "53882              Setapak      540000     4        2.0        3.0   \n",
+              "\n",
+              "             Property Type  Size (sqft)        Furnishing  \n",
+              "0       Serviced Residence       1335.0   Fully Furnished  \n",
+              "2              Condominium       1875.0  Partly Furnished  \n",
+              "4              Condominium       1513.0  Partly Furnished  \n",
+              "5                 Bungalow       7200.0  Partly Furnished  \n",
+              "7      Semi-detached House       3600.0  Partly Furnished  \n",
+              "...                    ...          ...               ...  \n",
+              "53862          Condominium        915.0  Partly Furnished  \n",
+              "53868          Condominium       1544.0   Fully Furnished  \n",
+              "53869          Condominium        650.0  Partly Furnished  \n",
+              "53876          Condominium       3973.0  Partly Furnished  \n",
+              "53882          Condominium       1313.0  Partly Furnished  \n",
```

Comment:
Can you please add a new column, Price/sqft. I think that will be helpful for machine learning process as we can put this value as independent value.

Reply:
still searching for the code. chatgpt pls help.